functions.php

```diff
@@ -409,6 +409,18 @@ function newspack_is_active_style_pack() {
 	return in_array( $active_style_pack, $args );
 }
 
+/**
+ * Check for specific templates.
+ */
+function newspack_check_current_template() {
+	global $post;
+
+	$post_id       = $post->ID;
+	$template_file = $post_id ? get_post_meta( $post_id, '_wp_page_template', true ) : '';
```

Comment:
When on Dashboard page with no post (e.g. `/wp-admin`) I get this PHP Warning:

` Notice: Trying to get property 'ID' of non-object in /srv/www/newspack-repos/newspack-theme/functions.php on line 418`

This can be corrected by removing  line 418 and changing this line to:

```suggestion
	$template_file = ( $post && $post->ID ) ? get_post_meta( $post->ID, '_wp_page_template', true ) : '';
```

Reply:
Ah, good catch, Jeff! This has been fixed in 440fd06 .